cogs/hideout/boost_roles.py

```diff
@@ -28,8 +27,16 @@ async def get_emoji(input: str) -> T:
 
         return await partial.read()
 
-    boost = app_commands.Group(name="boost", description="Commands for managing your boost.")
-    role = app_commands.Group(name="role", description="Commands for manging your boost role.", parent=boost)
+    @app_commands.guild_only()
+    class BoostGroup(app_commands.Group):
+        pass
+
+    @app_commands.guild_only()
+    class RoleGroup(app_commands.Group):
+        pass
+
+    boost = BoostGroup(name="boost", description="Commands for managing your boost.")
+    role = RoleGroup(name="role", description="Commands for manging your boost role.", parent=boost)
```

Comment:
boost = app_commands.Group(name="boost", description="Commands for managing your boost.", guild_only=True)
    @app_commands.guild_only()
    role = app_commands.Group(name="role", description="Commands for manging your boost role.", parent=boost)